src/resources/elements/tokenSelect/tokenSelect.html

```diff
@@ -7,7 +7,7 @@
       <div repeat.for="tokenInfo of tokenList">
         <div class="tokenItem">
           <img src.to-view="tokenInfo.logoURI"></img>
-          <div if.bind="!symbolOnly">${tokenInfo.name}</div>
+          <div if.bind="!symbolOnly">${tokenInfo.name.split('(')[0]}&nbsp;${tokenInfo.name.split('(').slice(1).join('(')}</div>
```

Comment:
this makes no sense to me, seems way over complicated.  All we need here is a space between the name and the symbol. that should require no more than adding `&nbsp;` after the token name or before the symbol.